### PR TITLE
upgrade to 3.1.0-preview4.19579.2

### PR DESCRIPTION
**Pull request by dieterdp**
upgrade to 3.1.0-preview4.19579.2, needed this for a project i am working on.

### CODE REVIEW

File: test/Blazor.Extensions.Logging.Test/wwwroot/index.html [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width">
-    <title>Blazor.Extensions.Logging.Test</title>
-    <base href="/" />
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width">
+  <title>Blazor.Extensions.Logging.Test</title>
+  <base href="/" />
+  <script src="_content/blazor.extensions.logging.js"></script>
```
Comment (chucker) [inline test/Blazor.Extensions.Logging.Test/wwwroot/index.html]: At least in my case, this needs to be `_content/blazor.extensions.logging/blazor.extensions.logging.js`.

(You can find the correct path by looking in `bin\…\YourProjectName.StaticWebAssets.xml`.)

Comment (umairsyed613) [reply test/Blazor.Extensions.Logging.Test/wwwroot/index.html]: when this PR will be completed and publish to nuget? current version doesnt work with netstandard2.1

Comment (Vaevictus) [reply test/Blazor.Extensions.Logging.Test/wwwroot/index.html]: bump, can we get this pushed to nuget?

**Comment by attilah** [comment]
Closing as package is update to Blazor RC1, available on [Nuget](https://www.nuget.org/packages/Blazor.Extensions.Logging/3.1.0-rc1-020501)

**Pull request closed without merging**